docs/deployment.md

```diff
@@ -2,10 +2,14 @@ TheCoin is (currently) a series of micro-services connected together.
 
 ## System setup require prior to deployment
 
-Each deployment app (rates/broker/site/nft) needs a GAE configuration.  The GAE configuration is used to set the deployment target for all the apps, deploying at the same time. See https://medium.com/google-cloud/how-to-use-multiple-accounts-with-gcloud-848fdb53a39a for how to create new configs.
+Each deployment app (rates/broker/site/nft) needs a GAE configuration for each environment (test/beta|prod).  The GAE configuration is used to set the deployment target for all the apps, deploying at the same time. See https://medium.com/google-cloud/how-to-use-multiple-accounts-with-gcloud-848fdb53a39a for how to create new configs.
```

Comment:
Might be a good idea to write what GAE means at least once.

Reply:
Done